torch/_torch_docs.py

```diff
@@ -1239,11 +1239,8 @@ def merge_dicts(*dicts):
 
         If :attr:`indices_or_sections` is a list of ints, :attr:`input` is split along
         dimension :attr:`dim` at each of the indices in the list. For instance,
-        :code:`[2, 3]` and :code:`dim=0` would result in the following tensors:
-
-            - :code:`input[:2]`
-            - :code:`input[2:3]`
-            - :code:`input[3:]`
+        :code:`[2, 3]` and :code:`dim=0` would result in the tensors :code:`input[:2]`,
```

Comment:
Maybe `indices_or_sections=[2, 3]` in the first code block? Your call. Just pint me with your thinking.

Reply:
Yeah that makes sense. Updated.